components/alert/alert.css

```diff
@@ -0,0 +1,28 @@
+@import url('../../starterFiles/abstracts/variables.css');
+@import url('../../starterFiles/base/reset.css');
+@import url('../../starterFiles//utils/util.css');
+@import url('../../starterFiles//utils/margin.css');
+@import url('../../starterFiles//utils/padding.css');
```

Comment:
you can create a main file for CSS and import all the other files into that including the Alert one